src/config.json

```diff
@@ -0,0 +1,130 @@
+{
+    "name": "Datadog",
+    "type": "executer",
+    "execProgram": "node",
+    "main": "app.js",
+    "exposeRoute": true,
+    "imgUrl": "datadog.jpeg",
+    "version": "1.0.0",
+    "description": "Handle Datadog REST API integration",
+    "settings": [
+      {
+        "name": "URL",
+        "viewName": "url",
+        "type": "string"
+      },  
+      {
+            "name": "API_KEY",
+            "viewName": "Api Key",
+            "type": "vault"
+        },
+        {
+            "name": "APP_KEY",
+            "viewName": "Application Key",
+            "type": "vault"
+        }
+    ],
+    "methods": [
+      {
+        "name":"CREATE_USER",
+        "viewName": "Create User",
+        "params": [
+          {
+            "name": "access_role",
+            "viewName" : "Access Role",
+            "type" : "options",
+            "options": [
+              {
+                "id":"st",
+                "name":"standard user"
+              },
+              {
+                "id":"adm",
+                "name":"admin user"
+              },
+              {
+                "id":"ro",
+                "name":"read only user"
+              }
+            ]
+          },
+          {
+            "name":"disabled",
+            "viewName": "disabled",
+            "type": "boolean"
+          },
+          {
+            "name": "email",
+            "viewName": "email",
+            "type" : "string"
+          },
+          {
+            "name": "handle",
+            "viewName": "handle email",
+            "type" : "string"
+          },
+          {
+            "name": "name",
+            "viewName": "name",
+            "type" : "string"
+          }
+          
+        ]
+      }, 
+      {
+            "name": "REST",
```

Comment:
@ilanyaniv please remove this method.
1. It doesn't exists in the code
2. It doesn't seem to have a dedicated purpose